profile/README.md

```diff
@@ -0,0 +1,24 @@
+<p align="center">
+  <img src="https://github.com/ipfs/.github/blob/main/profile/ipfs-powered.png" width="100%" alt="IPFS Powered">
```

Comment:
We should be able to use a relative link here, right? This would make MD previews work as expected.

```suggestion
  <img src="ipfs-powered.png" width="100%" alt="IPFS Powered">
```

Reply:
Thanks for checking :)